src/screens/track-screen/TrackScreenRemixes.tsx

```diff
@@ -0,0 +1,111 @@
+import { ID } from 'audius-client/src/common/models/Identifiers'
+import { View } from 'react-native'
+
+import IconArrow from 'app/assets/images/iconArrow.svg'
+import IconRemix from 'app/assets/images/iconRemix.svg'
+import Button, { ButtonType } from 'app/components/button'
+import { Tile } from 'app/components/core'
+import GradientText from 'app/components/gradient-text'
+import { flexRowCentered, makeStyles } from 'app/styles'
+import { useThemeColors } from 'app/utils/theme'
+
+import { TrackScreenRemix } from './TrackScreenRemix'
+
+const messages = {
+  title: 'Remixes',
+  viewAll: (count: number | null) =>
+    `View All ${count && count > 6 ? `${count} ` : ''}Remixes`
+}
+
+type TrackScreenRemixesProps = {
+  trackIds: ID[]
+  goToAllRemixes: () => void
+  count: number | null
+}
+
+const useStyles = makeStyles(({ palette, spacing, typography }) => ({
+  tile: {
+    padding: spacing(6)
+  },
+
+  header: {
+    ...flexRowCentered(),
+    justifyContent: 'center',
+    marginBottom: spacing(6)
+  },
+
+  headerText: {
+    lineHeight: 30,
+    fontSize: 28
+  },
+
+  iconRemix: {
+    height: spacing(12),
+    width: spacing(12),
+    marginRight: spacing(2)
+  },
+
+  tracks: {
+    flexDirection: 'row',
+    alignItems: 'flex-start',
+    justifyContent: 'center',
+    flexWrap: 'wrap'
+  },
+
+  track: {
+    marginHorizontal: spacing(3),
+    marginBottom: spacing(5)
+  },
+
+  buttonContainer: {
```

Comment:
there are a lot of custom styles here for buttons, wondering if these should be the defaults for our button component, or potentially the defaults for the "secondary" button type?

Reply:
This seems like the only place these specific styles are used. It might make sense to have a `small` button because I've seen that in other places, like the download button

Reply:
Word something we can do later to make button more flexible maybe, I'm also trying to update it with a new variant